dev/ColorPicker/readme.md

```diff
@@ -0,0 +1,141 @@
+
+* `ThirdDimensionSlider` : [ColorPickerSlider] The 1-dimensional slider representing the third channel of the selected color not shown in the `ColorSpectrum`. This will always be an HSV channel. 
+* `ThirdDimensionSliderGradientBrush` : [LinearGradientBrush] The image brush fill of the background rectangle that must be below the `ThirdDimensionSlider`. It visually represents a gradient of values the user can choose from. The background gradient rectangle itself is not a template part but should still be in the control template -- it just isn't used in code-behind. The background image brush is not managed by the slider but instead by the `ColorPicker`.
+* `AlphaSlider` : [ColorPickerSlider] The 1-dimensional slider representing the alpha channel of the selected color not shown in the `ColorSpectrum`.
+* `AlphaSliderGradientBrush` : [LinearGradientBrush] The image brush fill of the `AlphaSliderBackgroundRectangle` background rectangle that must be below the `AlphaSlider`. It visually represents a gradient of alpha values the user can choose from. The background image brush is not managed by the slider but instead by the `ColorPicker`.
+* `AlphaSliderBackgroundRectangle` : [Rectangle] The background rectangle whose fill is used to represent the gradient of values the user can choose from in the alpha slider. This is a template part -- unlike the third dimension -- because the checkered background must be re-rendered if its size changes.
+* `AlphaSliderCheckeredBackgroundImageBrush` : [ImageBrush] The image brush fill of the checkered background rectangle that must be below the `AlphaSlider`. The checkered background rectangle itself is not a template part but should still be in a control template -- it just isn't used in code-behind.
+
+* `MoreButton` : [ButtonBase] A button used to expand the color picker showing additional input controls.
+* `ColorRepresentationComboBox` : [ComboBox] The ComboBox used by the user to select between the RGB and HSV color representation. 
+* `RGBComboBoxItem` : [ComboBoxItem] The item specifying 'RGB' color representation within the `ColorRepresentationComboBox`.
+* `HSVComboBoxItem` : [ComboBoxItem] The item specifying 'HSV' color representation within the `ColorRepresentationComboBox`.
+
+* `RedTextBox` : [TextBox] The red channel input text box the user can modify.
+* `GreenTextBox` : [TextBox] The green channel input text box the user can modify.
+* `BlueTextBox` : [TextBox] The blue channel input text box the user can modify.
+* `HueTextBox` : [TextBox] The hue channel input text box the user can modify.
+* `SaturationTextBox` : [TextBox] The saturation channel input text box the user can modify.
+* `ValueTextBox` : [TextBox] The value channel input text box the user can modify.
+* `AlphaTextBox` : [TextBox] The alpha channel input text box the user can modify (shared for both RGB/HSV).
+* `HexTextBox` : [TextBox] The hexadecimal HTML color input text box the user can modify. This is always in RGB.
```

Comment:
Afaik hexadecimal colors are quite universal and not HTML specific. Also it's not always RGB but also ARGB if alpha is enabled.

Reply:
Yes, HTML isn't needed here

Reply:
RGB in this context means color representation not pixel format.

Reply:
Is there a HSL/HSV hexadecimal representation? Afaik hexadecimal always refers to a (A)RGB representation.

Reply:
I'm not sure if one is widely used. The problem arises with 0..359 for Hue being outside a byte. That said, there is no harm in being overly specific here I added it so no one would need to think about it.